pkg/cmd/roachtest/roachstress.sh

```diff
@@ -16,6 +16,66 @@ if [ -z "${BASH_VERSINFO}" ] || [ -z "${BASH_VERSINFO[0]}" ] || [ "${BASH_VERSIN
   exit 1
 fi
 
+function help {
+  cat <<EOF
+Stress run roachtest
+Usage:
+  $(basename ${0}) [flags] [-- roachtest flags]
+
+flags:
```

Comment:
Consider using a standard way of formatting these that also indicates whether they take an argument, e.g.:

```
-t TEST    test to run (overrrides TEST)
-c COUNT   number of tests to run (overrides COUNT)
-l         run test locally on host (overrides LOCAL)
...
```